perf.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	AllCPUs  = "-a"
+	Output   = "-o"
+	Detailed = "-d"
+	Event    = "-e"
+	SEP      = "-x"
+)
+
+// PerfStat A instance of perf stat command
+type PerfStat struct {
+	args    string
+	sep     string
+	outFile string
+	run     chan bool
+	stop    chan bool
+}
+
+// NewPerfStat returns a new instance for perf stat
+func NewPerfStat(args ...string) *PerfStat {
+	perfStat := new(PerfStat)
```

Comment:
Golang has native support for runtime arguments. Please rewrite.

Reply:
The idea is to create a new instance of perf command with a specific set of arguments rather than calling this function at run time.